src/sortStudents.ts

```diff
@@ -1,16 +1,64 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function avarageGrage(array: number[]): number {
+  return array.reduce((total, grade) => (total + grade), 0) / array.length;
+}
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+)
+  : Student[] {
+  const newStudents = [...students];
+
+  switch (sortBy) {
+    case 'name':
```

Comment:
use SortType enum here
```suggestion
    case SortType.Name:
```